src/main/java/teammates/common/datatransfer/InstructorPrivileges.java

```diff
@@ -350,7 +375,34 @@ public boolean isAllowedForPrivilege(String sectionName, String sessionName, Str
     public boolean isAllowedForPrivilegeAnySection(String sessionName, String privilegeName) {
         return isAllowedInSessionLevelAnySection(sessionName, privilegeName);
     }
-    
+
+    public boolean hasCoownerPrivileges() {
+        return hasSamePrivileges(coownerPrivileges);
+    }
+
+    public boolean hasManagerPrivileges() {
+        return hasSamePrivileges(managerPrivileges);
+    }
+
+    public boolean hasObserverPrivileges() {
+        return hasSamePrivileges(observerPrivileges);
+    }
+
+    public boolean hasTutorPrivileges() {        
+        return hasSamePrivileges(tutorPrivileges);
+    }
+
+    private boolean hasSamePrivileges(HashMap<String, Boolean> defaultPrivileges) {
+
+        for (String privilege : defaultPrivileges.keySet()) {
```

Comment:
Sorry, I missed this. Ok, let's open that up for discussion in a new issue. I think we can just use whatever we have here now, as long as it works.